fab/inventory/icds

```diff
@@ -21,12 +21,21 @@
 [es1]
 10.247.24.17 elasticsearch_node_name=es1 devices='["/dev/sdb","/dev/sdc","/dev/sdd","/dev/sde","/dev/sdf","/dev/sdg"]' partitions='["/dev/sdb1","/dev/sdc1","/dev/sdd1","/dev/sde1","/dev/sdf1","/dev/sdg1"]' datavol_device='/dev/mapper/consolidated-data' hot_standby_master='10.247.24.17'
 
+[es2]
+10.247.24.28 elasticsearch_node_name=es2 devices='["/dev/sdb"]' partitions='["/dev/sdb1"]' datavol_device='/dev/mapper/consolidated-data' hostname='es2'
+
+[es3]
+10.247.24.29 elasticsearch_node_name=es3 devices='["/dev/sdb"]' partitions='["/dev/sdb1"]' datavol_device='/dev/mapper/consolidated-data' hostname='es3' hot_standby_master='10.247.24.23'
+
 [couch0]
 10.247.24.18 devices='["/dev/sdb","/dev/sdc","/dev/sdd","/dev/sde","/dev/sdf"]' partitions='["/dev/sdb1","/dev/sdc1","/dev/sdd1","/dev/sde1","/dev/sdf1"]' datavol_device='/dev/mapper/consolidated-data'
 
 [celery0]
 10.247.24.19 devices='["/dev/sdb"]' partitions='["/dev/sdb1"]' datavol_device='/dev/mapper/consolidated-data'
 
+[celery1]
+10.247.24.31 devices='["/dev/sdb"]' partitions='[]' datavol_device='/dev/mapper/consolidated-data'
```

Comment:
This doesn't have any partitions, might need to double check ansible lvm to handle this